packages/extension-chakra-storefront/package.json

```diff
@@ -18,6 +18,7 @@
     "typecheck": "tsc --noEmit"
   },
   "devDependencies": {
+    "@ark-ui/react": "^5.8.0",
```

Comment:
Need to install @ark-ui so that it will be installed in the `node_modules` of extension-chakra-storefront, otherwise, moduleMapper will throw error since it is not existing